TelepresenceBot/mobile/src/main/res/layout/activity_bot.xml

```diff
@@ -1,6 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v7.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
-  android:id="@+id/bot_directions"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
   android:layout_height="match_parent"
-  android:scrollbars="vertical" />
+  android:orientation="vertical"
+  tools:context="com.novoda.tpbot.human.HumanActivity">
+
+  <android.support.v7.widget.RecyclerView
```

Comment:
I'm thinking of replacing this with the `DebugView` in my branch, it will make the process of building the debug version slightly faster.